var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -116,10 +132,17 @@ def cmake_args(self):
                 self.define_from_variant('Tasmanian_ENABLE_PYTHON', 'python'),
                 self.define_from_variant('Tasmanian_ENABLE_MPI', 'mpi'),
                 self.define_from_variant('Tasmanian_ENABLE_CUDA', 'cuda'),
+                self.define_from_variant('Tasmanian_ENABLE_HIP', 'rocm'),
```

Comment:
Is `rocm` also supported in earlier versions of the package? Or should this definition go under a conditional?

Reply:
I did this with a conflict:
```
conflicts('+rocm', when='@:7.3')
```

Reply:
Ok, just wanted to make sure that:
```
-DTasmanian_ENABLE_HIP:BOOL=OFF
```
don't cause any trouble with old versions.

Reply:
No, CMake is pretty chill about these things. After configuration, it will list unused variables in a short blurp but it isn't even a full warning.

Thanks for checking the PR.